### PR TITLE
feat(vue): Add Pinia Plugin Docs

### PR DESCRIPTION
**Pull request by onurtemizkan**
Documents: https://github.com/getsentry/sentry-javascript/pull/13841

### CODE REVIEW

**Comment by smeubank** [comment]
should we add to Nuxt as well? Pinia has more limited value add for a nuxt app but it can still be used there

File: docs/platforms/javascript/guides/vue/features/pinia.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: Pinia
+description: "Learn about Sentry's Pinia plugin."
+---
+
+To capture Pinia state data, use `Sentry.createSentryPiniaPlugin` and add it to your Pinia store instance.
```
Comment (coolguyzone) [inline docs/platforms/javascript/guides/vue/features/pinia.mdx:6]:
```suggestion
To capture Pinia state data, use `Sentry.createSentryPiniaPlugin()` and add it to your Pinia store instance.
```

Comment (coolguyzone) [inline docs/platforms/javascript/guides/vue/features/pinia.mdx:6]:
```suggestion
To capture Pinia state data, use `Sentry.createSentryPiniaPlugin()` and add it to your Pinia store instance:
```

**Review by coolguyzone — APPROVED** [review]
Added a minor suggestion, looks good, thanks for adding this! 👍

File: docs/platforms/javascript/guides/vue/features/pinia.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+});
+```
+
+## Options
+
+To configure the Pinia plugin, pass an options object as the first parameter to `createSentryPiniaPlugin`.
+
+<Alert level="warning" title="Note">
+
+While we try our best to filter out Personally Identifiable Information such as user passwords, we advise against sending sensitive information to Sentry.
```
Comment (s1gr1d) [inline docs/platforms/javascript/guides/vue/features/pinia.mdx:34]:
```suggestion
While we try our best to filter out Personally Identifiable Information (PII) such as user passwords, we advise against sending sensitive information to Sentry.
```

**Comment by s1gr1d** [comment]
To add those docs to the Nuxt docs, the plugin has to be added to the Sentry Nuxt SDK. I created an issue for that and will work on this and the docs for it: https://github.com/getsentry/sentry-javascript/issues/14039

**Pull request merged**